.gitattributes

```diff
@@ -0,0 +1 @@
+amostra/_version.py export-subst
```

Comment:
What's the role of this configuration?

Reply:
looks like this is for some kind of placeholder expansion https://git-scm.com/docs/gitattributes - but that only seems relevant when creating archives

Reply:
(removed, btw)